modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -211,6 +231,8 @@ API:: Specifies the API to show.
 
 The Application Analytics Graph component displays statistics about the use of an application, such as traffic, response time, and application health.
 
+Data shown from Anypoint Platform: Application statistics from Anypoint API Manager.
```

Comment:
Add link to API Manager docs homepage
Double check if this comes from API Manager or Monitoring center

Reply:
Link added. Still need to double check if this comes from API Manager or Monitoring center.

Reply:
Jesus confirmed today this is API Manager.